src/modules/launcher/Directory.Build.targets

```diff
@@ -2,12 +2,12 @@
   
    <Target Name="KillPowerLauncher" BeforeTargets="PreBuildEvent;BeforeClean">
     <Message Text="Killing process 'PowerLauncher.exe'" Importance="normal" ContinueOnError="true" />
-     <Exec Command="taskkill /F /IM PowerLauncher.exe"
+     <Exec Command="tasklist /FI &quot;IMAGENAME eq PowerLauncher.exe&quot; 2&gt;NUL | find /I /N &quot;PowerLauncher.exe&quot;&gt;NUL &#xD;&#xA; if %25ERRORLEVEL%25==0 ( taskkill /f /im PowerLauncher.exe )"
               IgnoreExitCode="true"
               IgnoreStandardErrorWarningFormat="true">
        <Output TaskParameter="ExitCode" PropertyName="ErrorCode"/>
      </Exec>
-     <Error Text="Cannot kill PowerLauncher.exe process." Condition="$(ErrorCode) == 1"/>
+     <Message Text="Cannot kill PowerLauncher.exe process." Condition="$(ErrorCode) == 1"/>
```

Comment:
I think this should still be an error.  Powerlauncher.exe is running, but for somereason taskkill didn't work.  I've seen this happen when i've got the powertoys installed from msi running powerlauncher, and i attempt to build any of the launcher projects.

Reply:
@ryanbodrug-microsoft I changed the declaration of this event a bit. In the earlier command the two separate error and message handling had to be added because in case of not finding the process it would throw an error if not handled. With the new command the behavior is such that the error is thrown only if it cannot kill the process for some reason. So for this case I removed the custom error message that we have added, and I let it throw normally in that case rather than having the conditional check on error codes. Another thing that I noticed was that the Exit code for this new command is always 1, so I had to leave the ignoreexitcode as it was earlier. I verified that build happens successfully without errors in both cases where launcher was running and launcher wasn't running. To repro the case where PowerLauncher.exe cannot be killed, the best scenario I found was starting PowerLauncher.exe as admin, since in that case the above taskkill should feel due to missing permissions. For that scenario this is the error message displayed:
`EXEC : error : The process "PowerLauncher.exe" with PID 15416 could not be terminated.` and when you build, at the end it shows up as an error.

Can you confirm if the above changes are fine? If so I'll merge this PR in.

Reply:
This works for me.  Thanks @arjunbalgovind ! :)